### PR TITLE
Adding Kved view with list and retrieve functions plus pagination

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: data_ocean/serializers/kved_serializers.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from rest_framework import serializers
 from data_ocean.models.kved_models import Kved
 
-class KvedSerializerWithModel(serializers.ModelSerializer):
+class KvedSerializer(serializers.ModelSerializer):
     class Meta:
         model = Kved
-        fields = ['code', 'name', 'group', 'division', 'section']
```
Comment (criticus) [inline data_ocean/serializers/kved_serializers.py:7]: don't we want to show group, division, section?

File: data_ocean/views/kved_views.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+from django.shortcuts import get_object_or_404
+from rest_framework import viewsets
+from rest_framework import generics
```
Comment (criticus) [inline data_ocean/views/kved_views.py:3]: can you squash rest_framework imports into one line?

File: data_ocean/views/kved_views.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+from rest_framework.response import Response
+from data_ocean.models.kved_models import Kved
+from data_ocean.serializers.kved_serializers import KvedSerializer
+from rest_framework.pagination import PageNumberPagination
```
Comment (criticus) [inline data_ocean/views/kved_views.py:7]: order imports alphabetically please

File: data_ocean/views/kved_views.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+class KvedView(viewsets.ReadOnlyModelViewSet, PageNumberPagination):
+    queryset = Kved.objects.all()
+    serializer_class = KvedSerializer
+
+    def list(self, request):
+        queryset = self.get_queryset()
+        results = self.paginate_queryset(queryset)
```
Comment (criticus) [inline data_ocean/views/kved_views.py:15]: where do we set pagination parameters? offset, limit?

File: data_ocean/serializers/kved_serializers.py (hunk 1 above)
Comment (IegorSoboliev) [reply data_ocean/serializers/kved_serializers.py:7]: Usually, only these two parameters are shown

File: data_ocean/views/kved_views.py (hunk 2 above)
Comment (IegorSoboliev) [reply data_ocean/views/kved_views.py:3]: done

File: data_ocean/views/kved_views.py (hunk 3 above)
Comment (IegorSoboliev) [reply data_ocean/views/kved_views.py:7]: done with automatic sorting

File: data_ocean/views/kved_views.py (hunk 4 above)
Comment (IegorSoboliev) [reply data_ocean/views/kved_views.py:15]: My idea is to do that in settings for all classes

File: data_ocean/serializers/kved_serializers.py (hunk 1 above)
Comment (criticus) [reply data_ocean/serializers/kved_serializers.py:7]: usually? what do you mean? It's our first service :)

File: data_ocean/views/kved_views.py (hunk 4 above)
Comment (criticus) [reply data_ocean/views/kved_views.py:15]: ok, please do so

File: data_ocean/serializers/kved_serializers.py (hunk 1 above)
Comment (IegorSoboliev) [reply data_ocean/serializers/kved_serializers.py:7]: I mean in all other sources)

Comment (IegorSoboliev) [reply data_ocean/serializers/kved_serializers.py:7]: And I hope this serializer will be used in other models. Two attributes are enough

File: data_ocean/views/kved_views.py (hunk 4 above)
Comment (IegorSoboliev) [reply data_ocean/views/kved_views.py:15]: done

File: data_ocean/serializers/kved_serializers.py (hunk 1 above)
Comment (IegorSoboliev) [reply data_ocean/serializers/kved_serializers.py:7]: I think the best solution is two have two serializers for Kved. Added new one with all attributes

File: data_converter/settings.py [hunk 5]
```diff
@@ -40,6 +40,8 @@
     'django.contrib.messages',
     'django.contrib.staticfiles',
     'rest_framework',
+    'rest_framework.authtoken',
+    'rest_auth',
```
Comment (criticus) [inline data_converter/settings.py:44]: I guess this should be in the next task)) but it's ok

**Pull request merged**